Cargo.toml

```diff
@@ -39,11 +40,15 @@ imagefmt = "4.0"
 smallvec = "0.4"
 specs = "0.9.1"
 rayon = "0.7"
+rodio = "0.5.1"
 ticketed_lock = "0.1"
 wavefront_obj = "5.0"
 
 thread_profiler = { version = "0.1", optional = true }
 
+[dev_dependencies]
```

Comment:
I guess it is.  Woops.  For some reason it still worked when I added this but I'll fix it.